### ISSUE TITLE
Pull request

### ISSUE THREAD

**Issue by aberezkin** [miklblitz/angular2-image-upload_uploaded#1]
Можно сделать пулл реквест в мой репозиторий. Но для начала нужно привести все в надлежащий вид, т.е. написать все на английском и т.п. Либо, если не хочешь заморачиваться, я могу просто у тебя списать и привести в порядок установленный в моем репозитории.

**Comment by miklblitz** [comment]
Да, давай лучше ты приведи в порядок, мне кажется у тебя лучше это получится. Изначально я пытался залить на твою ветку, что-то не получилось.

**Comment by miklblitz** [comment]
и еще, занимаюсь доработкой твоего плагина, не знаю после каждой успешной доработки тебе отправлять разницу кода? надо как-то решить этот вопрос.

**Comment by aberezkin** [comment]
Смотри, на гитхабе есть такая удобная штука, как пул реквест. Ты просто не правильно скопировал мой репозиторий, почитай вот я нашел пару статеек

[Как сделать pull request](https://github.com/urfu-2015/guides/blob/master/how-to-pull-request.md)
[Pull request'ы на GitHub или Как мне внести изменения в чужой проект](https://habrahabr.ru/post/125999/)
